src/renameKey.js

```diff
@@ -0,0 +1,36 @@
+import { always, curry, equals, when } from 'ramda';
+
+import renameKeysWith from './renameKeysWith';
+
+/**
+ * Creates a new object with the own properties of the provided object, but a
+ * single key is renamed according to the currentKey as `'oldKey'` and newKey as `'newKey'`.
+ * When some key is not found in the currentKey, then it's passed as-is.
+ *
+ * Keep in mind that in the case of keys conflict is behaviour undefined and
+ * the result may vary between various JS engines!
+ *
+ * @func renameKey
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.5.0|v1.5.0}
+ * @category Object
+ * @sig [String a] -> [String b] -> {[String a]: *} -> {[String b]: *}
```

Comment:
What would you say to this signature?
```suggestion
 * @sig (String a, String b) => a -> b ->  {a: *} -> {b: *}
```